testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -88,7 +88,7 @@ def parse_args():
     parser.add_argument("-v", "--version", dest="version",
                         help="Version of SUMA you want to run this script for, options are 42 for 4.2 or 43 for 4.3",
                         default="43", action='store')
-    parser.add_argument("-i", "--rrids", dest="rrids", help="RR IDs", default=None, action='store')
+    parser.add_argument("-i", "--mis", dest="mis", help="MI IDs", default=None, action='store')
```

Comment:
what about `mi_ids` and `mi_id` ?
Might be more readable.

Reply:
done